test/common/fmf_metadata/tests/cockpit_metadata_config.yaml

```diff
@@ -1,7 +1,6 @@
 additional_keys:
   _testlib__skipBrowser: "skipBrowser"
   _testlib__skipImage: "skipImage"
-  _testlib__skipPackage: "skipPackage"
```

Comment:
Yes, this was an abandoned attempt on making the tests more flexible for QE. But I think this got entirely replaced with TMT and gating tests everywhere now (as it should be). +1 for cleaning this up.

The same is true for the whole test/common/fmf_metadata/ directory, btw -- so if you feel like it, this could go as well.